### PR TITLE
add Historical Properties text

### PR DESCRIPTION
**Pull request by stpeter**

### CODE REVIEW

**Comment by stpeter** [comment]
This addresses #119

**Comment by becarpenter** [comment]
Works for me

**Comment by adamroach** [comment]
> proposals that might have a detrimental affect on these properties

Nit: effect

File: draft-iab-rfcefdp-rfced-model.md [hunk 1]
```diff
@@ -893,6 +893,56 @@ as follows:
 The third paragraph of the "Status of This Memo" section shall be
 as specified in {{Section 3.5 of RFC7841}}.
 
+# Historical Properties of the RFC Series {#properties}
+
+This section lists some of the properties that have been 
+historically regarded as important to the RFC Series. Proposals 
+that affect these properties are possible within the processes 
+defined in this document. As described under {{workflow}} and {{cfc}},
+proposals that might have a detrimental affect on these properties
```
Comment (stpeter) [inline draft-iab-rfcefdp-rfced-model.md:902]:
```suggestion
proposals that might have a detrimental effect on these properties
```

**Comment by adamroach** [comment]
LGTM

File: draft-iab-rfcefdp-rfced-model.md [hunk 2]
```diff
@@ -893,6 +893,56 @@ as follows:
...
+
+All existing RFC Series documents have been published in English. 
+Documents have been licensed to explicitly permit translation into 
+other languages.
+
+## Diversity
+
+In addition to Internet standards, the RFC series has published procedural 
+and informational documents, thought experiments, speculative ideas, 
+research papers, histories, humor, and even eulogies.
```
Comment (martinthomson) [inline draft-iab-rfcefdp-rfced-model.md:929]: JCK suggests (and I think this is better):

```suggestion
The RFC series has included many types of documents including standards for
the Internet, procedural and informational documents, thought experiments,
speculative ideas, research papers, histories, humor, and even eulogies.
```

File: draft-iab-rfcefdp-rfced-model.md [hunk 3]
```diff
@@ -893,6 +893,56 @@ as follows:
...
+consistent, and readable {{RFC7322}}.
+
+## Stability
+
+Once published, RFC Series documents have not changed.
+
+## Longevity
+
+RFC Series documents have been published in a form intended to be easily 
+legible to humans for decades or longer.
```
Comment (martinthomson) [inline draft-iab-rfcefdp-rfced-model.md:944]: JCK's suggestion:

```suggestion
accessible and usable to humans for decades or longer.
```

File: draft-iab-rfcefdp-rfced-model.md [hunk 4]
```diff
@@ -893,6 +893,56 @@ as follows:
...
+
+RFC Series documents have been published in a format that was intended 
+to be as accessible as possible to those with special needs, e.g., for 
+those with impaired sight.
+
+## Language
+
+All existing RFC Series documents have been published in English. 
+Documents have been licensed to explicitly permit translation into 
+other languages.
```
Comment (martinthomson) [inline draft-iab-rfcefdp-rfced-model.md:923]: A lightly edited version of @adamroach's suggestion:

```suggestion
Documents have been made available under terms that allow anyone
to translate them into other languages without asking for permission.
```

**Review by martinthomson — COMMENTED** [review]
Capturing some of the discussion thus far on list.  The actionable parts anyway.  Some requires more discussion, I sense.

File: draft-iab-rfcefdp-rfced-model.md [hunk 5]
```diff
@@ -893,6 +893,56 @@ as follows:
...
+proposals that might have a detrimental effect on these properties
+should receive heightened scrutiny during RSWG discussion and RSAB 
+review. The purpose of this scrutinty is to ensure that all changes are 
+deliberate and that the consequences of a proposal, as far as they can be
+identified, have been carefully considered.
+
+## Availability
+
+Documents in the RFC Series have been available for more than 35 years, 
+with no fee for access.
```
Comment (martinthomson) [inline draft-iab-rfcefdp-rfced-model.md:911]: JCK mentions that the series is >50yo.  This is true, but it's not necessary to be so precise:

```suggestion
Documents in the RFC Series have been available for many decades, 
with no fee for access.
```

File: draft-iab-rfcefdp-rfced-model.md [hunk 6]
```diff
@@ -893,6 +893,57 @@ as follows:
...
+consistent, and readable {{RFC7322}}.
+
+## Stability
+
+Once published, RFC Series documents have not changed.
+
+## Longevity
+
+RFC Series documents have been published in a form intended to be easily 
+accessible and usable to humans for decades or longer.
```
Comment (martinthomson) [inline draft-iab-rfcefdp-rfced-model.md:945]: Based on discussion on-list, a thought:

```suggestion
comprehensible to humans for decades or longer.
```

File: draft-iab-rfcefdp-rfced-model.md [hunk 7]
```diff
@@ -893,6 +893,57 @@ as follows:
...
+RFC Series documents have been published in a format that was intended 
+to be as accessible as possible to those with special needs, e.g., for 
+those with impaired sight.
+
+## Language
+
+All existing RFC Series documents have been published in English. 
+However, since the beginning of the RFC series, translation into a 
+language other than English has been allowed and encouraged. The IETF 
+Trust and the RFC Editor place no restrictions on this.
```
Comment (martinthomson) [inline draft-iab-rfcefdp-rfced-model.md:924]:
```suggestion
language other than English has been allowed and encouraged.
```

File: draft-iab-rfcefdp-rfced-model.md [hunk 8]
```diff
@@ -893,6 +893,57 @@ as follows:
...
+proposals that might have a detrimental effect on these properties
+should receive heightened scrutiny during RSWG discussion and RSAB 
+review. The purpose of this scrutiny is to ensure that all changes are 
+deliberate and that the consequences of a proposal, as far as they can be
+identified, have been carefully considered.
+
+## Availability
+
+Documents in the RFC Series have been available for many decades, 
+with no fee for access.
```
Comment (martinthomson) [inline draft-iab-rfcefdp-rfced-model.md:911]:
```suggestion
with no constraints on access or distribution.
```

**Review by martinthomson — COMMENTED** [review]
A few more thoughts.

File: draft-iab-rfcefdp-rfced-model.md (hunk 8 above)
Comment (martinthomson) [reply draft-iab-rfcefdp-rfced-model.md:911]: or
```suggestion
with no constraints on access or redistribution.
```

File: draft-iab-rfcefdp-rfced-model.md (hunk 6 above)
Comment (martinthomson) [reply draft-iab-rfcefdp-rfced-model.md:945]: Stephen points out that pairing that word with "easily" might not be wise.  For some reason I can't re-suggest here (GitHub failure), so maybe consider that also.

**Review by martinthomson — APPROVED** [review]

**Pull request merged**